lib/vtls/openssl.c

```diff
@@ -2467,6 +2468,11 @@ static CURLcode ossl_connect_step1(struct connectdata *conn, int sockindex)
   }
 #endif
 
+#ifdef HAVE_SSL_CTX_SET_POST_HANDSHAKE_AUTH
+   /* OpenSSL 1.1.1 requires clients to opt-in for PHA */
+   SSL_CTX_set_post_handshake_auth(BACKEND->ctx, 1);
```

Comment:
Just a minor nit: Indentation is off-by-one to the right.

Reply:
Argh, fixed!